scripts/openstack.py

```diff
@@ -0,0 +1,156 @@
+import argparse
+import sys
+
+import teuthology.openstack
+
+
+def main(argv=sys.argv[1:]):
+    teuthology.openstack.main(parse_args(argv), argv)
+
+
+def parse_args(argv):
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description="""
+Run a suite of ceph integration tests. A suite is a directory containing
+facets. A facet is a directory containing config snippets. Running a suite
+means running teuthology for every configuration combination generated by
+taking one config snippet from each facet. Any config files passed on the
+command line will be used for every combination, and will override anything in
+the suite. By specifying a subdirectory in the suite argument, it is possible
+to limit the run to a specific facet. For instance -s upgrade/dumpling-x only
+runs the dumpling-x facet of the upgrade suite.
+
+Display the http and ssh access to follow the progress of the suite
+and analyze results.
+
+  firefox http://183.84.234.3:8081/
+  ssh -i teuthology-admin.pem ubuntu@183.84.234.3
+
+""")
+    parser.add_argument(
+        '-v', '--verbose',
+        action='store_true', default=None,
+        help='be more verbose',
+    )
+    parser.add_argument(
+        '--name',
+        help='OpenStack primary instance name',
+        default='teuthology',
+    )
+    parser.add_argument(
+        '--key-name',
+        help='OpenStack keypair name',
+        required=True,
+    )
+    parser.add_argument(
+        '--key-filename',
+        help='path to the ssh private key',
+    )
+    parser.add_argument(
+        '--simultaneous-jobs',
+        help='maximum number of jobs running in parallel',
+        type=int,
+        default=2,
+    )
+    parser.add_argument(
+        '--integration-tests',
+        action='store_true', default=None,
+        help='run integration tests when creating the cluster',
+    )
+    parser.add_argument(
+        '--teardown',
+        action='store_true', default=None,
+        help='destroy the cluster, if it exists',
+    )
+    # copy/pasted from scripts/suite.py
```

Comment:
We could if both shared the same argument parsing method. However the variance is very low and updating options in two places when one changes every three month is probably not a big burden.